Makefile

```diff
@@ -92,22 +92,22 @@ compile:
 .PHONY: test
 test:
 	@echo "### Testing code"
-	GOOS=$(GOOS) go test -mod vendor -a ./... -coverpkg=./... -coverprofile cover.out
+	GOOS=$(GOOS) go test -mod vendor -a ./... -coverpkg=./... -coverprofile cover.all.out
```

Comment:
Did you try to adapt the coverpkg option here?

In theory we should be able to exclude unwanted files here and directly have a clean output.

Reply:
Yeah I tried, but `coverpkg` only allows you picking up a single package and not a bunch of packages, nor just excluding selected packages.

Reply:
Ok, thanks!